doc/README.md

```diff
@@ -6,7 +6,6 @@
 - [AirTable](/doc/AIRTABLE.md)
 - [Heroku](#heroku)
 - [CircleCI](#circleci)
-- [BrowserStack](#browserstack)
```

Comment:
Was this an accident? We still have a section in this doc for browserstack

Reply:
Line #9 and #11 == duplicates. Hence the removal.

Reply:
Oh of course, good catch!